samples/subsys/ipc/openamp_rsc_table/README.rst

```diff
@@ -18,73 +18,110 @@ a Linux kernel OS on the main processor and a Zephyr application on
 the co-processor.
 
 Building the application
-*************************
+************************
 
 Zephyr
--------
+------
```

Comment:
This should actually be `=` not `-` as per https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#headings -- would be great if you could perform the mass replace :)

Reply:
Thanks for the review, done !